MahjongAI/MajsoulClient.cs

```diff
@@ -826,9 +833,26 @@ private string GetDeviceUUID()
         private string getServerHost(string serverListUrl)
         {
             var webClient = new WebClient();
-            var serverListJson = webClient.DownloadString(Constants.MAJSOUL_API_URL_PRIFIX[config.MajsoulRegion] + serverListUrl);
+            var gateUrl = Constants.MAJSOUL_API_URL_PRIFIX[config.MajsoulRegion].Item1;
+            var serverListJson = webClient.DownloadString(gateUrl + serverListUrl);
             var serverList = JObject.Parse(serverListJson)["servers"];
             return (string)serverList[0];
+        } 
+        
+        private Dictionary<string, string> getServerVersion()
```

Comment:
A tuple is better than a dictionary in this use case. https://devblogs.microsoft.com/dotnet/new-features-in-c-7-0/#tuples

Is it supported by the current target framework version? If not, feel free to change the version.

Reply:
We can add separate library System.ValueTuple https://www.nuget.org/packages/System.ValueTuple/  without update target framework version.